test/jdk/javax/swing/text/CSSBorder/6796710/bug6796710.java

```diff
@@ -113,7 +125,10 @@ public void run() {
         // On Linux platforms realSync doesn't guaranties setSize completion
         Thread.sleep(1000);
 
-        if (!Util.compareBufferedImages(bufferedImage, getPnBottomImage())) {
+        BufferedImage pnBottomImage = getPnBottomImage();
+        if (!Util.compareBufferedImages(bufferedImage, pnBottomImage)) {
+            ImageIO.write(bufferedImage, "png", new File("bufferedImage"));
```

Comment:
I guess .png extension needs to be added to new File()

Reply:
> I guess .png extension needs to be added to new File()
Fixed.